### ISSUE TITLE
Nautobot healthchecks document for celery-beat isn't clear that ENVVAR NAUTOBOT_CELERY_BEAT_HEARTBEAT_FILE needs to be set

### ISSUE THREAD

**Issue by gneville-ot** [nautobot/nautobot#5549]
### As ...

Patti - Platform Admin

### I want ...

A message within the Nautobot healthchecks document regarding celery-beat that in order to use the healthcheck command listed here: https://github.com/nautobot/nautobot/blob/e1f02ea094d0a484711a2ef06a2f7b37c2a0c9bd/nautobot/docs/user-guide/administration/installation/health-checks.md#nautobot-celery-beat, the NAUTOBOT_CELERY_BEAT_HEARTBEAT_FILE ENVVAR must be present.

### So that ...

To avoid missing a step and troubleshooting why healthchecks fail.

### I know this is done when...

The documentation is updated so it is clear that NAUTOBOT_CELERY_BEAT_HEARTBEAT_FILE must be present for the healthcheck to work.

### Optional - Feature groups this request pertains to.

- [ ] Automation
- [ ] Circuits
- [ ] DCIM
- [ ] IPAM
- [ ] Misc (including Data Sources)
- [ ] Organization
- [ ] Plugins (and other Extensibility)
- [ ] Security (Secrets, etc)
- [ ] Image Management
- [ ] UI/UX
- [X] Documentation
- [ ] Other (not directly a platform feature)

### Database Changes

No

### External Dependencies

None

**Comment by gneville-ot** [comment]
I ran in to this when troubleshooting why on Kubernetes the healthcheck kept failing. It wasn't clear that `NAUTOBOT_CELERY_BEAT_HEARTBEAT_FILE` must be set for the check to work.

**Comment by glennmatthews** [comment]
Good catch. Probably it would be best to default the check to the path `/tmp/nautobot_celery_beat_heartbeat` if the variable is unset, rather than failing.

**Comment by gneville-ot** [comment]
In addition it might be worth adding something indicating that the `-mmin -0.1` should be changed if the periodSeconds of the exec command is changed from the example of 5. 

In my environment I can't run exec commands that frequent and had to alter accordingly.
